Python/Product/VSInterpreters/Interpreter/CondaEnvironmentManager.cs

```diff
@@ -60,15 +60,16 @@ public static CondaEnvironmentManager Create(string condaPath) {
             return new CondaEnvironmentManager(condaPath);
         }
 
+        // Creates a new conda environment with the given name or path, installing the specified packages. 'conda create -n name python pkg1 pkg2 ... -y'
         public async Task<bool> CreateAsync(string newEnvNameOrPath, IEnumerable<PackageSpec> packageSpecs, ICondaEnvironmentManagerUI ui, CancellationToken ct) {
             bool success = false;
             using (await _working.LockAsync(ct)) {
                 var args = new[] {
                     "create",
                     IsAbsolutePath(newEnvNameOrPath) ? "-p" : "-n",
                     ProcessOutput.QuoteSingleArgument(newEnvNameOrPath),
-                    "-y",
-                }.Union(packageSpecs.Select(s => s.FullSpec));
+                    "python",                    
```

Comment:
since conda 22, python isn't automatically added the new env folder.

"python" here means use the latest.